package.json

```diff
@@ -25,6 +25,7 @@
   "homepage": "https://github.com/digitalbazaar/bedrock-tokenization",
   "dependencies": {
     "assert-plus": "^1.0.0",
+    "bitstring": "^1.0.0",
```

Comment:
This is not our bitstring.  Ours is `@digitalbazaar/bitstring` and you import it using the same name.

Reply:
Opps, sorry. I'll change it, thanks.